test/src/org/omegat/filters/MoodlePHPFilterTest.java

```diff
@@ -38,10 +38,10 @@
     public void testParse() throws Exception {
         parse(new MoodlePHPFilter(), "test/data/filters/MoodlePHP/file.php");
     }
-
+    
```

Comment:
A lot of garbage whitespace was added to this file. Could you remove them please?

Reply:
Solved.